src/common/kits/BackgroundEventManager.ts

```
@@ -1,29 +1,102 @@
  * @description background事件管理器
  */
 class BackgroundEventManager {
   static instance: BackgroundEventManager | null = null;
 
+  connectList: {
+    // uuid: string;
+    port: chrome.runtime.Port;
+  }[] = [];
+
   static getInstance(): BackgroundEventManager {
     if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)